frontend/src/actions/adminActions.tsx

```diff
@@ -50,16 +40,23 @@ export interface IRole {
   id?: number;
 }
 export interface IAccessRequest {
```

Comment:
probably make sense to move this out as well.

Reply:
done